### DISCUSSION TITLE
SP-API "InternalFailure" responses all a sudden this morning. Anyone else?

### DISCUSSION THREAD

**Discussion by thebest0f** [amzn/selling-partner-api-models#3441] [General] [answered]
The GET ORDERS end point started failing this morning, on and off. 

Anyone else experiencing this issue?

**Comment by fegbers** [comment, marked as answer]
FYI: https://sellercentral.amazon.com/sp-api-status

**Reply by thebest0f** [reply]
Thanks! Ill bookmark that

**Comment by getvicky83** [comment]
its fixed, i can see all green in europe, In northamerica only one red in catalog api

**Comment by Dragonsangel** [comment]
What is the correct way of monitoring Up/Down-time of the API?

There was a post in Discussions, notifying us about the issue amzn/selling-partner-api-docs#2763. But in the past, that message was just edited to say that the issue is resolved, for which we do not receive notifications. 
E-Mail notifications were also not sent out for this occurrence, potentially since most of the APIs affected do not have an active MWS-API any more.
The SP-API Status page helps to see if there is currently an issue, but outside writing an HTML-Scraper to look for changes on that page, there is no RSS-Feed or anything similar to automatically notify observers that there is a status change.

We need to notify our clients when there is a problem downstream from us, and this is currently a very manual process, which is not quite optimal.

**Reply by sungolivia** [reply]
@Dragonsangel  agreed, our team is actually aware of this and we are working on it in fact! Will let you know once there is an update.
